### PR TITLE
Pass pod spec lint

### PR DESCRIPTION
**Pull request by morganchen12**
I haven't tested integrating this into an app, but this PR allows the built framework from `build.swift` and its accompanying podspec to pass lint. Proof-of-concept for #32.

### CODE REVIEW

**Comment by 1amageek** [comment]
Thank you!!
I am very glad 😆😆😆😆😆😆

File: Salada.podspec [hunk 1]
```diff
@@ -1,37 +1,20 @@
...
+  s.summary                 = "Salada is OR Mapper for Firebase."
+  s.homepage                = "https://github.com/1amageek/Salada"
+  s.license                 = { :type => 'BSD', :file => 'SaladaFrameworks/LICENSE' }
+  s.author                  = "1amageek"
+  s.social_media_url        = "https://twitter.com/1amageek"
+  s.platform                = :ios
+  s.ios.deployment_target   = "8.0"
+  s.ios.framework           = "UIKit"
+  s.requires_arc            = true
+  s.source                  = { :http => "https://github.com/1amageek/Salada/releases/download/2.1/SaladaFrameworks.zip" }
```
Comment (morganchen12) [inline Salada.podspec:14]: @1amageek In order for `pod spec lint` to pass you should upload `SaladaFrameworks.zip` to this URL and use CocoaPods >= `1.2.1`.

Comment (1amageek) [reply Salada.podspec:14]: OK
- Cocoapods upgraded
- Salada `Frameworks.zip` uploaded.

https://github.com/1amageek/Salada/releases/tag/2.1

```
pod lib lint
 -> Salada (2.0)
Salada passed validation.
```

Comment (1amageek) [reply Salada.podspec:14]: It seems there is a problem with the nested classes defined in Salada
I will fix it.

**Pull request merged**